common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_common.yaml

```diff
@@ -146,6 +146,9 @@ sClientAssociationParams:
   bssid: sMacAddr
   capabilities: beerocks::message::sRadioCapabilities
   vap_id: int8_t
+  association_frame:
+    _type: char
+    _length: [ "beerocks::message::ASSOCIATION_MAX_LENGTH" ] 
```

Comment:
You really should use a dynamic length here, because there is no 0 terminator to indicate the length. But then it can't be a struct any more :-( But actually, this struct really serves no purpose at all, it can be inlined directly into cACTION_APMANAGER_CLIENT_ASSOCIATED_NOTIFICATION. So please do that (in a separate commit) and then add the association frame as a dynamic length field.

Note that in the 1905 message it's an unknown length field, but that's not possible here because we don't have a total length in the beerocks classes.